llvmlite/binding/module.py

```diff
@@ -240,6 +240,24 @@ def _next(self):
         return ffi.lib.LLVMPY_FunctionsIterNext(self)
 
 
+class _BlocksIterator(_Iterator):
+
+    def _dispose(self):
+        self._capi.LLVMPY_DisposeBlocksIter(self)
+
+    def _next(self):
+        return ffi.lib.LLVMPY_BlocksIterNext(self)
```

Comment:
`LLVMPY_BlocksIterNext` is missing the type declaration for its `.restype`. it is being assumed to be `ctypes.c_int`.  You will need to add `ffi.lib. LLVMPY_BlocksIterNext.restype = ffi.LLVMValueRef`.

Reply:
Fixed.